site/templates/film/item.jet

```diff
@@ -18,21 +19,31 @@
 {{end}}
 
 {{block body()}}
+
+{{sponsorText := film.CustomFields.GetString("sponsor_image_text_film", "")}}
+{{sponsorLink := film.CustomFields.GetString("sponsor_image_link_film", "")}}
+{{sponsorLinkIsExternal := film.CustomFields.GetBool("sponsor_image_link_external_film", false)}}
+{{sponsorImage := film.ImageMap["Sponsor"]}}
```

Comment:
There variables need an indent if they are staying here.  But IMO they would be less intrusive if they lived in the sponsor-image jet.  To do that you would need to pass in the film as context instead of using these individual parameters.  Thoughts?

Reply:
Changed - Yeah that makes the template look much nicer